news/2022-08-04-wiki-contributors-wanted.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title: "Wiki contributors wanted!"
+date: 2022-08-04 10:00:00 +0000
+---
+
+Lore maestros! Technical aficionados! Information brokers! We seek any kind of knowledge to share in the wiki, and to translate it for every corner of the community.
+
+![](/wiki/shared/news/banners/osu-wiki.jpg)
+
+The [osu! wiki](/wiki/Main_Page) has seen a lot of reconstruction and development over the years. However, while the to-do list grows larger with new ideas and things to expand on, the amount of contributors has remained very small, which has impeded any rapid progress. We are therefore looking for more people to help out!
```

Comment:
```suggestion
The [osu! wiki](/wiki/Main_Page) has seen a lot of reconstruction and development over the years. However, while the to-do list grows larger with new ideas and things to expand on, the amount of contributors has remained very small. We are therefore looking for more people to help out!
```
it doesn't seem necessary to say this part, kind of implied by contrasting the large "to-do list" with small # of contributor. but mainly I just think this paragraph has a better rhythm without extra clause here